### PR TITLE
Rock64 U-Boot Image

### PR DESCRIPTION
**Pull request by CameronNemo**
This image uses the latest upstream sources and no binary blobs.
I have tested the boot on my device.

### CODE REVIEW

File: srcpkgs/rock64-uboot/template [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+do_configure() {
+	make rock64-rk3328_defconfig
+}
+
+do_build() {
+	unset CFLAGS CXXFLAGS CPPFLAGS LDFLAGS
+	if [ "$CROSS_BUILD" ]; then
+		export CROSS_COMPILE="${XBPS_CROSS_TRIPLET}-"
+	fi
+	if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
```
Comment (Piraty) [inline srcpkgs/rock64-uboot/template:26]: https://github.com/void-linux/void-packages/pull/24307

Comment (CameronNemo) [reply srcpkgs/rock64-uboot/template]: removed

File: srcpkgs/rock64-uboot/template [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Template file for 'rock64-uboot'
+pkgname=rock64-uboot
+version=2020.07
+revision=1
+archs="aarch64*"
+wrksrc="u-boot-${version}"
+build_style=gnu-makefile
```
Comment (Piraty) [inline srcpkgs/rock64-uboot/template:7]: that's obsolete if you provide all of the `do_*()` functions, no?

**Review by Piraty — APPROVED** [review]

File: srcpkgs/rock64-uboot/template (hunk 2 above)
Comment (CameronNemo) [reply srcpkgs/rock64-uboot/template:7]: yep thanks for pointing that out, removed the line

File: srcpkgs/rock64-uboot/template [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+revision=1
+archs="aarch64*"
+wrksrc="u-boot-${version}"
+hostmakedepends="flex bc python3 swig python3-devel dtc"
+makedepends="atf-rk3328-bl31"
+short_desc="Das U-Boot for the Rock64 SBC"
+maintainer="Cameron Nemo <cnemo@tutanota.com>"
+license="GPL-2.0-or-later, BSD-3-Clause"
+homepage="https://www.denx.de/wiki/U-Boot/"
+distfiles="ftp://ftp.denx.de/pub/u-boot/u-boot-${version}.tar.bz2"
```
Comment (sgn) [inline srcpkgs/rock64-uboot/template:13]: I think we can fetch it via `https`?

Comment (CameronNemo) [reply srcpkgs/rock64-uboot/template:13]: oh true, I will adjust that then. (and for #21199).

Comment (CameronNemo) [reply srcpkgs/rock64-uboot/template]: done here, will update the pbp PR at a later date

**Pull request merged**